src/test/java/sirius/biz/storage/layer2/BlobHardRefSpec.groovy

```diff
@@ -0,0 +1,40 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.storage.layer2
+
+import sirius.db.jdbc.OMA
+import sirius.kernel.BaseSpecification
+import sirius.kernel.di.std.Part
+
+class BlobHardRefSpec extends BaseSpecification {
+
+    @Part
+    private static OMA oma;
+
+    @Part
+    private static BlobStorage blobStorage;
+
+    def "store entity with unchanged hard ref blob must not fail"() {
```

Comment:
while being at it - maybe add another test for the happy path (meaning the given part of the test below as when: and in then: simply ensure that the blob is no longer temporary)

and then delete the entity and check that the blob is marked as deleted :) (either in one case or in two cases)

Reply:
done